files/default/cloudwatch_logs/cloudwatch_log_files.json

```diff
@@ -48,27 +48,6 @@
       ],
       "feature_conditions": []
     },
-    {
-      "timestamp_format_key": "default",
-      "file_path": "/var/log/cfn-init-cmd.log",
```

Comment:
Might need to revisit in the future to see which `cfn-init` log to keep if we were to add timestamps to track individual recipe's execution time. Ok for now.